<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <property name="spacing">
     <number>3</number>
    </property>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_2">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_3">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTableWidget" name="tableWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <column>
       <property name="text">
        <string>prot</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>local_address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>rem_address</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>state</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>pid</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>program</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>cmdline</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QFrame" name="frame">
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Filter:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionRestart_as_Root"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <widget class="QMenu" name="menuUpdate_speed">
     <property name="title">
      <string>&amp;Update speed</string>
     </property>
     <addaction name="action1_second"/>
     <addaction name="action3_second"/>
     <addaction name="action5_second"/>
     <addaction name="actionPause"/>
    </widget>
    <addaction name="menuUpdate_speed"/>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <widget class="QMenu" name="menuProt">
     <property name="title">
      <string>&amp;Prot</string>
     </property>
     <addaction name="actionTCP"/>
     <addaction name="actionUDP"/>
     <addaction name="actionTCP6"/>
     <addaction name="actionUDP6"/>
    </widget>
    <widget class="QMenu" name="menuState">
     <property name="title">
      <string>&amp;State</string>
     </property>
     <addaction name="actionESTABLISHED"/>
     <addaction name="actionLISTEN"/>
     <addaction name="actionCLOSE"/>
     <addaction name="actionAllOther"/>
    </widget>
    <addaction name="actionUtilite_for_kill"/>
    <addaction name="actionResolve_Addresses"/>
    <addaction name="menuProt"/>
    <addaction name="menuState"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>&amp;Refresh Now</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="action1_second">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;1 second</string>
   </property>
   <property name="shortcut">
    <string>Alt+1</string>
   </property>
  </action>
  <action name="action3_second">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;3 second</string>
   </property>
   <property name="shortcut">
    <string>Alt+2</string>
   </property>
  </action>
  <action name="action5_second">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;5 second</string>
   </property>
   <property name="shortcut">
    <string>Alt+3</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Pause</string>
   </property>
   <property name="shortcut">
    <string>Alt+0</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>&amp;Save As...</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
  </action>
  <action name="actionRestart_as_Root">
   <property name="text">
    <string>&amp;Restart as Root</string>
   </property>
  </action>
  <action name="actionResolve_Addresses">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Resolve Addresses</string>
   </property>
  </action>
  <action name="actionTCP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;TCP</string>
   </property>
  </action>
  <action name="actionUDP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;UDP</string>
   </property>
  </action>
  <action name="actionTCP6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>T&amp;CP6</string>
   </property>
  </action>
  <action name="actionUDP6">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>U&amp;DP6</string>
   </property>
  </action>
  <action name="actionCLOSE">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;CLOSE</string>
   </property>
  </action>
  <action name="actionESTABLISHED">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;ESTABLISHED</string>
   </property>
  </action>
  <action name="actionLISTEN">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;LISTEN</string>
   </property>
  </action>
  <action name="actionAllOther">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;All other</string>
   </property>
  </action>
  <action name="actionUtilite_for_kill">
   <property name="text">
    <string>&amp;Utilite for kill...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
